src/main/resources/global/89-init-alfresco-json-logging.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+echo "Alfresco json logging init start"
+
+if [[ $ALFRESCO_JSON_LOGGING == true ]]
+then
+    JSON_LOGGING_VERSION=${JSON_LOGGING_VERSION:-'0.0.3'}
+    # simulate the deployment of the amp from https://github.com/xenit-eu/alfred-json-logging
+    # assumes the necessary jars are present in /docker-config/dependencies
+    cp /docker-config/dependencies/* /usr/local/tomcat/webapps/alfresco/WEB-INF/lib/
+
+    mkdir -p /usr/local/tomcat/webapps/alfresco/WEB-INF/classes/alfresco/module/alfred-json-logging/
+    echo "<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE beans PUBLIC '-//SPRING//DTD BEAN//EN' 'http://www.springframework.org/dtd/spring-beans.dtd'>
+<!-- Workaround for MNT-18557: force the log4j hierarchy to be initialized as early as possible. -->
+<beans>
+  <bean id=\"eu.xenit.alfred.logging.NullBeanDefinitionRegistryPostProcessor\" class=\"eu.xenit.alfred.logging.NullBeanDefinitionRegistryPostProcessor\" depends-on=\"log4JHierarchyInit\"/>
+</beans>" >/usr/local/tomcat/webapps/alfresco/WEB-INF/classes/alfresco/module/alfred-json-logging/module-context.xml
+    echo "module.version=${JSON_LOGGING_VERSION}
+module.description=Alfresco json logging
+module.id=alfred-json-logging
+module.repo.version.max=999
+module.title=alfred-json-logging
+module.repo.version.min=0
+module.installState=INSTALLED
+module.installDate="`date +"%FT%T.%3N%:z" |sed 's/:/\\\\:/g'` >/usr/local/tomcat/webapps/alfresco/WEB-INF/classes/alfresco/module/alfred-json-logging/module.properties
```

Comment:
what does the intention of the `sed` instruction exactly ?

Reply:
to append a "\" in front of ":" (correctly-escaped properties file):

module.installDate=2019-11-14T15\:17\:45.146+00\:00